crates/store/src/db/sql/mod.rs

```diff
@@ -944,14 +943,24 @@ pub fn unconsumed_network_notes(
         notes.push(NoteRecord::from_row(row)?);
         // Increment by 1 because we are using rowid >=, and otherwise we would include the last
         // element in the next page as well.
-        token.0 = row.get::<_, i64>(11)? + 1;
+        token.0 = row.get::<_, u64>(11)? + 1;
     }
 
     Ok((notes, token))
 }
 
 #[derive(Default, Debug, Copy, Clone)]
-pub struct PaginationToken(i64);
+pub struct PaginationToken(u64);
```

Comment:
nit: this seems like a pretty general struct. I'd probably move it to the top of the file and add doc comments for it.